shared/src/main/scala/io/lemonlabs/uri/Authority.scala

```diff
@@ -110,6 +122,8 @@ object Authority {
   implicit val orderAuthority: Order[Authority] = Order.by { authority =>
     (authority.userInfo, authority.host, authority.port)
   }
+
+  val DefaultPorts: Map[String, Int] = Map("http" -> 80, "https" -> 443)
```

Comment:
Since RFC3986 doesn't seem to specify a set of default ports, maybe we could add these others from the WHATWG URL spec. (I'm planning for scala-uri fully support the WHATWG spec in the future anyway)

scheme | port
-- | --
"ftp" | 21
"http" | 80
"https" | 443
"ws" | 80
"wss" | 443